### ISSUE TITLE
Default project when the GNS3 client is started

### ISSUE THREAD

**Issue by 1001QAdotNET** [GNS3/gns3-server#1629]
In the eding project dialog add an option to open this project (if the server could be contacted)
![image](https://user-images.githubusercontent.com/17384308/62406718-0ff91680-b57e-11e9-951e-d04523795c00.png)

**Comment by grossmj** [comment]
Please, can you explain this a bit more? thanks.

**Comment by 1001QAdotNET** [comment]
Currently the client is crashing like hell when I delete nodes. I add them using the apis
this behaviour It is not very consistent. I used CSR1000v and it would crash no matter what if I delete them from the GUI
If I use the APIs the client (which i still use to visualize the changes) is more tolerant but sometimes it crashes this way too
Then I changed the IOS to IOSv and deleting nodes added via APIs would not cause a client (GUI) crash. After a while after playing with my scripts the client started crashing even if I delete one node sometimes, sometimes I manage to delete a couple of them it crashes and then i have to restart the client like 2 times to delete all 20 nodes in my topology
Every time when I go through these cycles I need to go and pick the project I want to open. I would prefer to have an option to open the project that I chose automatically (have a checkmark somewhere where it says -this is the default project I want to open when the client comes up)
